cmd/controller/app/controller.go

```diff
@@ -134,13 +138,13 @@ func buildControllerContext(opts *options.ControllerOptions) (*controller.Contex
 			Recorder:                        recorder,
 			KubeSharedInformerFactory:       kubeSharedInformerFactory,
 			SharedInformerFactory:           sharedInformerFactory,
-			Namespace:                       opts.Namespace,
+			Namespace:                       defaultNamespace,
```

Comment:
The namespace field can be removed from this structure altogether afaik? I don't think it's used.

Reply:
I was thinking about the last part of the original issue that talks about potentially adding the flag back in the future, so left it around. It'd be cleaner to just remove it though.